Pod/Classes/ViewControllers/ARFairBMWArtActivationComponentViewController.m

```diff
@@ -0,0 +1,29 @@
+#import "ARFairBMWArtActivationComponentViewController.h"
+#import "AREmission.h"
+#import "ARGraphQLQueryPreloader.h"
+
+@interface ARFairBMWArtActivationComponentViewController ()
+
+@end
+
+@implementation ARFairBMWArtActivationComponentViewController
+
++ (NSArray<ARGraphQLQuery *> *)preloadQueriesWithFairID:(NSString *)fairID;
```

Comment:
We can remove this method from this file and the header, actually. It's only really used in a few situations and doesn't really apply here.